experiments/example/main.go

```diff
@@ -139,8 +137,15 @@ func main() {
 					"qps":          qps,
 					experiment.ExperimentKey: uid,
 				}
+
+				output, err := mutilateTask.StdoutFile()
+				errutil.CheckWithContext(err, fmt.Sprintf("Cannot get Mutilate output file"))
+
+				// Create Mutilate Snap session launcher - it will be used to gather metrics about Memcached performance.
+				mutilateSnapSession, err := mutilatesession.NewSessionLauncherDefault(output.Name(), tags)
```

Comment:
@skonefal - `err` seems not to be checked here.

Reply:
Done.